app/views/layouts/application.html.haml

```diff
@@ -6,6 +6,8 @@
     = csrf_meta_tags
     = stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+    #map{style: 'width: 800px; height: 400px'}
```

Comment:
So is the map showing here? Then it will be shown on every page, right? Should it not be only on the index page, or am I missing something? :)

Reply:
nope it should just be on main page, good catch. Thanks

Reply:
Woho!

Reply:
Thanks for the insight